db/seeds/permissions.rb

```diff
@@ -5,6 +5,7 @@ class PermissionsPopulator
     "application.non_passported.employment.*" => "Can create, edit, delete employment applications",
     "application.non_passported.bank_statement_upload.*" => "Can upload bank statements",
     "application.full_section_8.*" => "Can use full set of section 8 proceedings",
+    "application.non_passported.enhanced_bank_upload.*" => "Can use enhanced bank upload journey",
```

Comment:
if we do implement such a role i would be tempted to make it a sub role

```suggestion
    "application.non_passported.bank_statement_upload.enhanced.part_1*" => "Can use enhanced bank upload journey",
```
or similar. 

My thinking is they relate to bank_statement_upload only and there are further enhancements to be consider in the future potentially

Reply:
🤔 or should we be using a flag instead?!

Reply:
We already have the enhanced bank upload flag, and the feature sits behind that.

The more I'm thinking about it, the more it feels like the feature flag will be sufficient on its own.

We probably need a product steer on this, but I'd imagine when we turn this feature on, we will want to turn it on for all providers at firms with existing bank upload permissions. 

@RoseSAK what do you think?

Reply:
If we do want to use firm permissions though, I agree that it makes sense for this role to be sub-role of `application.non_passported.bank_statement_upload`.